budget_management_project/expense/urls.py

```diff
@@ -1,6 +1,10 @@
 from django.urls import path
-from . import views
+
+from expense.views import ExpenseCreateView, ExpenseUpdateView
 
 app_name = "expense"
 
-urlpatterns = []
+urlpatterns = [
+    path('api/v1/expenses', ExpenseCreateView.as_view(), name='expense-create'),
+    path('api/v1/expenses/<int:pk>', ExpenseUpdateView.as_view(), name='expense-update'),
```

Comment:
```suggestion
    path('api/v1/expenses/<int:expense_id>', ExpenseUpdateView.as_view(), name='expense-update'),
```
가 가독성에 좋아 보여 수정해주시면 감사하겠습니다 :)